media-video/syncplay/syncplay-1.4.0_rc2.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+inherit python-r1
+
+MY_PV=${PV/_rc/-RC}
+
+DESCRIPTION="Client/server to synchronize media playback"
+HOMEPAGE="http://syncplay.pl"
+SRC_URI="https://github.com/Syncplay/syncplay/archive/v${MY_PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+client +server gui vlc"
+REQUIRED_USE="vlc? ( client )
+	gui? ( client )"
+
+DEPEND=""
+RDEPEND="${PYTHON_DEPS}
+	dev-python/twisted-core[${PYTHON_USEDEP}]
+	gui? ( dev-python/pyside[${PYTHON_USEDEP}] )
+	vlc? ( media-video/vlc[lua] )"
+
+S="${WORKDIR}/${PN}-${MY_PV}"
+src_prepare() {
+	sed -i -e '22s%/usr%${PREFIX}%' -e '23s%/usr%%' -e '33,36s%${PREFIX}$(BASE_PATH)%${DESTDIR}${PREFIX}%' GNUmakefile || die "could not sed GNUmakefile"
```

Comment:
@mgorny The makefile stayed the same for two years. I'll submit the changes upstream, so I hope it won't be needed at all on the next release (upstream has planned to add support for PREFIX and DESTDIR for a while)